app/containers/DisplayWalletAccounts/DisplayWalletAccountsQrCodes/DisplayWalletAccountsQrCodes.jsx

```diff
@@ -95,6 +100,14 @@ class DisplayWalletAccountsQrCodes extends Component<Props, State> {
                 }}
               />
             </div>
+            <div className={styles.qr}>
+              <label> encrypted key </label>
+              <canvas
+                ref={node => {
+                  this.encryptedCanvas = node
+                }}
+              />
+            </div>
```

Comment:
Nice job here! Would like it if we could add padding around the div containing the QR codes... It looks a little a little off without any. I think it would be nice to also add the ability for users to copy the encrypted key qr to clipboard like they can for the public address qr
<img width="688" alt="Screen Shot 2019-06-16 at 7 22 10 PM" src="https://user-images.githubusercontent.com/13072035/59572755-7b3d6680-906c-11e9-9de3-a4d01cc2dad5.png">

Reply:
In regards to the padding, Could you set the proper measure? I could not tell whether it is better using the padding or not. :sweat_smile:

Reply:
After taking a closer look I think something like this will work just fine

<img width="663" alt="Screen Shot 2019-06-18 at 7 00 54 PM" src="https://user-images.githubusercontent.com/13072035/59725387-807ceb80-91fb-11e9-8341-96a124ca1ce9.png">
<img width="228" alt="Screen Shot 2019-06-18 at 7 01 02 PM" src="https://user-images.githubusercontent.com/13072035/59725389-807ceb80-91fb-11e9-9bfc-469297d1943b.png">


Also lets remove the ability to copy the private key code to the clipboard (keep that as it is currently in `master`)